tests/test_main.py

```diff
@@ -34,6 +41,33 @@ def read_profilometer(self, samples=1):
         prof_val -= 5000
         return prof_val
 
+    def set_tank_pressure(self, desired_pressure):
+        g = self.g
+        timeout = 90
+        time_passed = 0
+
+        g.write('M125 S' + str(desired_pressure))
+        # Wait for pressurization
+        while (True):
+            resp = g.write('M105', resp_needed=True)
+            # String returned is of the form:
+            # ok T:25.0 /0.0 B:0.0 /0.0 T0:25.0 /0.0 T1:0.0 /0.0 P:35.7 /0.0...
+            if (float(resp.split('P:')[1].split()[0]) >= desired_pressure):
+                break
+            elif (time_passed >= timeout):
+                print('ERROR: Timeout while waiting for pressurization')
```

Comment:
Do you want the test to fail in this case? If so `raise` this instead of printing it:

```python
raise RuntimeError('Timeout while waiting for pressurization')
```